src/plugins/data/public/ui/settings/settings.ts

```diff
@@ -35,7 +35,7 @@ export class Settings {
     private readonly queryEnhancements: Map<string, QueryEnhancement>,
     private readonly queryEditorExtensionMap: Record<string, QueryEditorExtensionConfig>
   ) {
-    this.isEnabled = this.config.enabled;
+    this.isEnabled = true;
```

Comment:
Why is this always true?

Reply:
ah that actually makes sense not to be true good catch.